app/lib/linters/crunch_api.rb

```diff
@@ -0,0 +1,97 @@
+require 'rest-client'
+require 'base64'
+module Linters
+
+  # A class called Crunch
+  class CrunchApi
+
+    attr_accessor :base_url
+
+    def initialize()
+      @base_url = "https://platform.42crunch.com"
+    end
+
+    # Lints an unploaded file with 42crunch.
+    #
+    # @param file [File] the file to be linted
+    # @return [String] the raw JSON response
+    def lint_to_json(file)
+
+      # Initialize local variables
+      create_response = String.new
```

Comment:
If we want to go down this route we could try sorbet? Means we can specify types.

Reply:
The usual way to define a new String in ruby is to use `""`. So:
```ruby
create_response = ""
```
Also, why do these variables need to initiated here anyway? They are all assigned values within the code below, and by doing it this way you end up with doing:
```ruby
create_response += RestClient.post( @base_url + "/api/v1/apis"
```
When this would be a lot simpler:
```ruby
create_response = RestClient.post( @base_url + "/api/v1/apis"
```